### PR TITLE
Nigel

### PR DESCRIPTION
**Pull request by Ruhtarduhd**
Create order. cause everybody likes order. #nochaos.

### CODE REVIEW

**Comment by Ruhtarduhd** [comment]
screen shot 2018-04-12 at 10 53 53
screen shot 2018-04-12 at 10 52 54

File: src/containers/internalContactOverview.js [hunk 1]
```diff
@@ -136,43 +324,61 @@ class InternalContactOverview extends PureComponent {
             onLeftIconButtonClick={_=> this.setState({drawer: !this.state.drawer})}
           />
           <Avatar
-            src='https://www.watsonmartin.com/wp-content/uploads/2016/03/default-profile-picture.jpg'
+            src= {this.state.currentUser.picture}
```
Comment (Safisyx) [inline src/containers/internalContactOverview.js]: Are we really going to have a profile picture of a user? 
Or we are just going to add a default picture

Comment (Ju1111) [reply src/containers/internalContactOverview.js]: I removed this avatar now. Think it's not needed

Comment (Ju1111) [reply src/containers/internalContactOverview.js]: as in now is in the code I am working on that is not pushed to github yet 😆

File: src/App.js [hunk 2]
```diff
@@ -25,11 +26,12 @@ class App extends Component {
...
-          <Route exact path="/login" component={LoginPage} />
-          <Route exact path="/signup" component={SignupPage} />
-          <Route exact path="/InternalContactOverview" component={InternalContactOverview} />
-          <Route exact path="/forgotpassword" component={ ForgotPassword }/>
-          <Route exact path="/flexicon/users/:usersId" component={ContactPage}/>
+          <Route exact path="/login" component={ LoginPage } />
+          <Route exact path="/signup" component={ SignupPage } />
+          <Route exact path="/create/order" component={ OrderCreator } />
+          <Route exact path="/create/InternalContactOverview" component={ InternalContactOverview } />
+          <Route exact path="/flexicon/forgotpassword" component={ ForgotPassword }/>
```
Comment (Ju1111) [inline src/App.js]: this should be without flexicon

File: src/App.js [hunk 3]
```diff
@@ -25,11 +26,12 @@ class App extends Component {
...
         <div className="App">
-          <Route exact path="/login" component={LoginPage} />
-          <Route exact path="/signup" component={SignupPage} />
-          <Route exact path="/InternalContactOverview" component={InternalContactOverview} />
-          <Route exact path="/forgotpassword" component={ ForgotPassword }/>
-          <Route exact path="/flexicon/users/:usersId" component={ContactPage}/>
+          <Route exact path="/login" component={ LoginPage } />
+          <Route exact path="/signup" component={ SignupPage } />
+          <Route exact path="/create/order" component={ OrderCreator } />
+          <Route exact path="/create/InternalContactOverview" component={ InternalContactOverview } />
```
Comment (Ju1111) [inline src/App.js]: and the two create/ routes should be flexicon/create/...

File: src/containers/internalContactOverview.js [hunk 4]
```diff
@@ -136,43 +324,61 @@ class InternalContactOverview extends PureComponent {
...
-            marginBottom: 5,
-          }}/>
-          <Divider />
-          {companies.map((company) => (
-            <div>
-            <ListItem
-              primaryText={`${company.companyName}`}
-              leftAvatar={<Avatar src={`${company.companyLogo}`} />}
-              rightIcon={<Business />}
+        <Paper
```
Comment (Ju1111) [inline src/containers/internalContactOverview.js]: The app bar can be deleted from here

File: src/App.js (hunk 3 above)
Comment (Safisyx) [reply src/App.js]: Why should we have some with flexicon/ and some not?

Comment (Ju1111) [reply src/App.js]: because the navbar is a component that will be on all the pages with /flexicon

Comment (Safisyx) [reply src/App.js]: oooh cool! :smile:

**Pull request merged**